src/index.css

```diff
@@ -60,32 +60,50 @@ code {
   }
 }
 
+/* Dropzone styles */
+
 .dropzone {
```

Comment:
Why are the dropzone styles here?

Reply:
It was already there. Will move them to the correct place